package com.simpli.exception1;

import java.util.Scanner;

public class ExceptionMain1 {

	public static void divide(int num, int den) {

		int res = num / den;

		System.out.println("Result =" + res);
	}

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		System.out.println("Enter the numerator :");
		int num = sc.nextInt();

		System.out.println("Enter the denomirator :");
		int den = sc.nextInt();

		try { // Inspect the Exception
			divide(num, den);
		} catch (ArithmeticException ex) { // Handle the Exception

			System.out.println("Denominator should not be zero !");
		}

		System.out.println("Rest code..........");

	}

}
